app/javascript/packages/document-capture/components/in-person-full-address-search.tsx

```diff
@@ -169,7 +184,7 @@ function FullAddressSearch({
   const handleSearch = useCallback(
     (event) => {
       onError(null);
-      onSearch(event, addressValue, cityValue, stateValue, zipCodeValue);
+      onSearch(event, addressValue.trim(), cityValue.trim(), stateValue, zipCodeValue);
```

Comment:
I only used the trim method on address and city. The value of state is coming from a drop down (not a user input) and removal of white space is already being handled in the inputChangeHandlerForZipCode method.